guides/plugins/plugins/framework/flow/action-transactions.md

```diff
@@ -0,0 +1,86 @@
+---
+nav:
+  title: Running actions inside transactions
+  position: 30
+
+---
+
+# Action transactions
+
+## Overview
+
+In this guide, you'll learn how to run your action code inside a transaction. This may be important for you if you want to graciously handle rollbacks in certain scenarios. We have implemented various abstractions to ease this process; however, you need to opt in.
```

Comment:
```suggestion
In this guide, you will learn how to run your action code inside a transaction. This may be important for you if you want to graciously handle rollbacks in certain scenarios. We have implemented various abstractions to ease this process; however, you need to opt in.
```